Snowdrift.cabal

```diff
@@ -215,9 +215,9 @@ library
                  , time
                  , titlecase
                  , transformers
-                 , unix
```

Comment:
Nope, check this: https://github.com/yesodweb/yesod-scaffold/blob/master/yesod-scaffold.cabal — no unix needed, and we didn't add it I believe (but I should check to be sure), anyway, I think it's fine to remove; also incidentally, warp *is* included in the scaffolding and we accidentally / unnecessarily removed it, so it is fine to put back in.